packages/cli/test/functional/test_site/sub_site/nested_sub_site/site.json

```
@@ -21,7 +19,7 @@
       "glob": "**/index.md"
     },
     {
-      "glob": "**/*.+(md|mbd)"
+      "glob": "**/*.+(md)"
```

I believe this can be simplified to
```suggestion
      "glob": "**/*.md"
```
Same for the other glob pattern.